Thumbprint/Targets/Thumbprint/Components/UserAvatar.swift

```diff
@@ -105,17 +107,14 @@ public final class UserAvatar: UIView {
     }
 
     private let badgeView: OnlineBadgeView
-    private var avatarHeightConstraint: Constraint?
 
     private func setupViews() {
         avatar.label.text = initials
         avatar.clipsToBounds = true
 
         addSubview(avatar)
         avatar.snp.makeConstraints { make in
-            make.top.left.equalToSuperview()
-            avatarHeightConstraint = make.height.equalTo(size.dimension).constraint
-            make.width.equalTo(avatar.snp.height)
+            make.edges.equalToSuperview()
```

Comment:
is there a reason we used the size constraint before ?

Reply:
I think it was just buggy. The author was setting the height of the avatar, but not constraining the bottom to self (or setting self.height).  This was causing the avatar to be larger than self.  The change I made was to pin the avatar's edges to self, and use intrinsicContentSize to set the size of self.